Myb 249 gis service refactor

safink opened the pull request:
Final step of Services refactor. Just added a new URLRequestConvertible enumeration inside BaseNetworkingService with the paths needed.

Commit:
MYB-249: GisService Refactor

-Added keys in AppConfiguration plist for GIS and methods in Configuration class
-Created GisRouter enumeration and migrated GisService functionality to BaseNetworkingService
-Refactored method calls in GisService class

Commit:
Merge branch 'sprint-9' into MYB-249-GisServiceRefactor

lisandro commented on MyBus/MyBus/Services/BaseNetworkService.swift:
```diff
@@ -40,13 +40,66 @@ enum GeoCodingRouter:URLRequestConvertible{
+                    params["method"] = GISRouter.MUNICIPALITY_WS_METHOD
+                    params["endpoint"] = "callealtura_coordenada" //callealtura o calleaaltura (?)
+                    params["token"] = GISRouter.MUNICIPALITY_ACCESS_TOKEN
+                    //address??
```
Seba, endpoint string you used is right and for _address_ param we need to send:
- [ ] `codigocalle` param
- [ ] `altura` param

Extracted request params from API docs [here](https://s9.postimg.org/kdzo5t54f/Screenshot_2016_08_11_19_51_36.png)

Commit:
Merge branch 'sprint-9' into MYB-249-GisServiceRefactor

Commit:
MYB-249-GisServiceRefactor

Updated AddressToCoordinate case method in GisRouter in BaseNetworkingService:
- Added GET params "codigocalle" y "altura"
